### ISSUE TITLE
Feat: More informations on Contributors Profiles

### ISSUE THREAD

**Issue by ceeriil** [elizaOS/elizaos.github.io#16]
It would be helpful to improve the contributors' profile page by adding  links that make it easier to  visit their Github profiles and view their contributions in detail. Clicking the contributions on the profile should redirect to github for further details

## Related Issues
[GitHub Issue #7](https://github.com/elizaOS/elizaos.github.io/issues/7)

**Comment by madjin** [comment]
Agree, right now can't click to see what PRs were made and each profile page could use a personal summary that can be generated based on contributions

![Image](https://github.com/user-attachments/assets/eb370f8a-1a37-4955-8a54-3914439c0104)

**Comment by madjin** [comment]
I think we accomplished this with substantial improvements, can make a new issue for improvements
